images/openstack/Dockerfile.ci

```diff
@@ -11,6 +11,13 @@ COPY --from=builder /go/src/github.com/openshift/installer/images/openstack/rdo-
 COPY --from=builder /go/src/github.com/openshift/installer/images/openstack/rdo-stein.gpg /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-Cloud
 COPY --from=registry.svc.ci.openshift.org/origin/4.1:cli /usr/bin/oc /bin/oc
 
+# Install Dependendencies for tests
+# https://github.com/openshift/origin/blob/master/images/tests/Dockerfile.rhel
```

Comment:
If this is copied from origin, link to state at a specific commit (not `master`) to help folks who have to make future tracking bumps.

Reply:
Done